LookingMenu/Utils/IdStoryBoardViews.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+final class IdStoryBoardViews {
+    static let welcomeVC = "Welcome_VC"
```

Comment:
```suggestion
    static let welcomeVC = "WelcomeVC"
```
tương tự với các vc dưới. _ chỉ đối với các keyDefault thôi e